app/controllers/jobs_controller.rb

```diff
@@ -3,6 +3,12 @@ class JobsController < ApplicationController
 
   def index
     @jobs = Job.all
+    @applied_jobs = Job.where(status: 'Applied')
+    @interested_jobs = Job.where(status: 'Interested')
+    @interviewed_jobs = Job.where(status: 'Interviewed')
+    @offered_jobs = Job.where(status: 'Offered')
+    @rejected_jobs = Job.where(status: 'Rejected')
+    @no_status_jobs = Job.where(status: nil)
```

Comment:
this is definitely one way to approach this. 

Another way would be to just keep only `@jobs` and then in the view do: 

```
@jobs.select {|job| job.status == "Applied"} 
```

Personally, I prefer the latter but I think your way is totally fine too - I'm just commenting it to make you aware of another way

Reply:
A third (and probably better) way to do this would be to keep only @jobs and then in the view do:

```
@jobs.select {|job| job.at_stage_applied? }
```

an then in the `models/job.rb` you can define that method

```
class Job
  # job model code - removing for brevity
  def at_stage_applied?
    status == "Applied"
   end
end
```

Reply:
ah okay, i might play around witht this